Update zoonotic disease section (#1138)
greenelab/covid19-review: content/09.evolution.md

As a result, the risk of zoonotic disease increases when there is substantial interaction between humans and wildlife [@doi:10.1016/j.pt.2016.04.007].
Many factors can influence this human/wildlife interface and therefore the risk of zoonotic transmission events [@doi:10.1016/j.pt.2016.04.007; @doi:10.3201/eid1112.040789].

In the SARS epidemic, SARS-CoV-1 was also thought to have emerged in a live animal market.
A survey of a market in Shenzen, China revealed that individuals from two carnivore species, namely several masked palm civets (_Paguma larvata_) and one racoon dog (_Nyctereutes procyonoides_), were likely carriers of SARS-CoV-1, despite presenting as healthy [@doi:10.1126/science.1087139].
However, further analysis suggested that these species might be only intermediate hosts who were exposed in the market setting [@doi:10.7150/ijbs.45472].
A closely related virus was identified in Chinese horseshoe bats (_Rhinolophus sinicus_), but the sequence identity was only 88% with SARS-CoV-1 [@doi:10.1073/pnas.0506735102].
Therefore, the species of origin for SARS-CoV-1 remains unresolved.

> In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle].

Reviewer: ```suggestion
In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle], where it would later emerge that many animals were sold suffering from poor health and hygiene [@doi:10.1038/s41598-021-91470-2].
```

This evidence therefore suggested that the virus may have emerged as a result of zoonotic transfer of a virus from bats to humans, with the wildlife trade considered a potential source of exposure.

Nevertheless, some fragments of the genome differ between SARS-CoV-2 and RATG13 by up to 17%, suggesting a complex natural selection process.
Additionally, SARS-CoV-2 is closely related (91.02%) to a novel coronaviruses identified in Malayan pangolins (_Manis javanica_) infected with a respiratory disease in October 2019 [@doi:10.1016/j.cub.2020.03.022].
Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (_S_) gene with SARS-CoV-2 drew further attention [@doi:10.1038/s41586-020-2169-0; @doi:10.1016/j.cub.2020.03.022].
The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change [@doi:10.1016/j.cub.2020.03.022], and the sequence identity between the regions is 97.4% [@doi:10.1038/s41586-020-2169-0].
Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.

> However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].

Reviewer: It is possible that a bats infect animals associated with these markets thereby increasing the likelihood of finding a bat-derived virus at one of these markers

Author: ```suggestion
However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
Additionally, endemic bat species are typically in hibernation at the time that SARS-CoV-2 emerged [@doi:10/ggjr43].
```

Reviewer: Perfect

> Therefore, it is not clear whether the disease emerged in a different location and/or whether it is associated with a different species.

Reviewer: Super!

There were 38 species observed at the market in the 2.5 years leading up to the emergence of SARS-CoV-2, indiciating significant diversity in the animals with which humans were interacting [@doi:10.1038/s41598-021-91470-2].
As with SARS-CoV-1, the species of origin for SARS-CoV-2 therefore remains unresolved.

Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin _S_ has resulted in popular conspiracy theories that the virus did not arise naturally.
The similarity of _S_ to that of pangolin viruses could arise from either recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153], rather than requiring human intervention.

> Such suspcions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].

Reviewer: ```suggestion
Such suspicions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].
```

> This ambiguity therefore serves to highlight the value of monitoring zoonotic diseases, as more robust reference panels against which to compare SARS-CoV-2 would be valuable in pinpointing its origin.

Reviewer: I don't know what you think about this - but my thought is we shouldn't shy away from the fact there is political motivations here too to promote a lab leak theory,  that we don't need to delve into, but should briefly acknowledge.

Author: Maybe something like:

```suggestion
While critiques that more thorough investigation of the origins of COVID-19 would be valuable may have some merit, in many cases, claims made in support of the "lab-leak" theory are politically motivated [@doi:10.1038/d41586-021-01383-3].
A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of monitoring zoonotic diseases.
```

Reviewer: Sorry I wasn't sure of the wording, you might want to edit my suggestion @rando2

Reviewer: Did you mean critics?

Author: I made a few changes, what do you think of this? And I don't think I meant to say critics anywhere but I'm not quite sure what part you're referring to, sorry @RLordan!

Reviewer: Good with me! No worries, just to let you know that I edited out the word "critiques" from the line that starts "While it has been suggested that more ..."

However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin _S_ has resulted in popular conspiracy theories that the virus did not arise naturally.
The similarity of _S_ to that of pangolin viruses could arise from either recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153], rather than requiring human intervention.
Such suspicions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].
While it has been suggested that more thorough investigation of the origins of COVID-19 may have some value [@doi:10.1126/science.abj0016], in many cases, support for the "lab-leak" theory is politically motivated [@doi:10.1038/d41586-021-01383-3].

> A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.

Author: ```suggestion
In general, approaching the origination of SARS-CoV-2 to develop proactive approaches to manage future emerging pathogens would be expected to provide more value than a reactive interest in a pathogen that has already caused a pandemic.
A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
```

Author: @RLordan thinking about this, I feel like maybe it's worth pointing out that proactive interest in the origins of SARS-CoV-2 is good, reactive blame-games are not where we need to be investing our energy or this will happen again. As I keep telling my father-in-law, I would be delighted if people genuinely cared about and valued biosafety standards, but that isn't the vibe I get from the articles he sends me :laughing:

Reviewer: I totally agree with you, I have my dad in my family to send me those perspectives haha 

I would agree on those perspectives, there is too much fingerprinting, and not enough logic. How do you write that politely haha
